lib/src/proto/info.pbserver.dart

```diff
@@ -1,7 +1,8 @@
 ///
 //  Generated code. Do not modify.
 //  source: info.proto
-///
-// ignore_for_file: non_constant_identifier_names,library_prefixes,unused_import
+//
+// @dart = 2.3
```

Comment:
Is this line autogenerated too?

Reply:
Yes: https://github.com/dart-lang/protobuf/blob/13b27f542e099cf1891746d18c88e7c013c94cc7/protoc_plugin/lib/file_generator.dart#L548